src/main/resources/application-local.yml

```
@@ -0,0 +1,29 @@
+
+mybatis:
+  mapper-locations: classpath:mapper/*.xml
+  configuration:
+    map-underscore-to-camel-case: true
+
+jwt:
+  secret: ${JWT_SECRET}
+  expiration: 3600000
+
+file:
+  upload-dir: ${FILE_UPLOAD_DIR}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)